### PR TITLE
Fix/dbml alias & primary key & note content bug

### PR DESCRIPTION
**Pull request by Huy-DNA**
## Summary
* Fix error message when alias is a duplicate.
* Allow alias to be the same as the same table's name
* Interpret `primary key`
* No longer normalize note content, left as-is.

## Issue
(issue link here)

## Lasting Changes (Technical)

* Additional check in `TableValidator`.
* Additional logic in `TableInterpreter`.
* Remove `normalizeNoteContent`.
* Testcases for these.

## Checklist

Please check directly on the box once each of these are done

- [ ] Documentation (if necessary)
- [X] Tests (integration test/unit test)
- [ ] Integration Tests Passed
- [ ] Code Review

### CODE REVIEW

File: packages/dbml-parse/src/lib/analyzer/validator/elementValidators/table.ts [hunk 1]
```diff
@@ -422,3 +426,14 @@ function isValidColumnType(type: SyntaxNode): boolean {
...
+
+function isAliasSameAsName(alias: string, nameFragments: string[]): boolean {
+  if (nameFragments.length > 1 || nameFragments.length === 0) {
+    return false;
+  }
+  if (alias === nameFragments[0]) {
+    return true;
+  }
+
+  return false;
```
Comment (NQPhuc) [inline packages/dbml-parse/src/lib/analyzer/validator/elementValidators/table.ts]:
```suggestion
  return nameFragments.length === 1 && alias === nameFragments[0];
```

File: packages/dbml-parse/src/lib/analyzer/validator/elementValidators/table.ts [hunk 2]
```diff
@@ -134,7 +134,7 @@ export default class TableValidator implements ElementValidator {
 
     const maybeNameFragments = destructureComplexVariable(name);
     if (maybeNameFragments.isOk()) {
-      const nameFragments = maybeNameFragments.unwrap();
```
Comment (NQPhuc) [inline packages/dbml-parse/src/lib/analyzer/validator/elementValidators/table.ts:137]: why change this em? are we going to mutate the name fragements?

Comment (Huy-DNA) [reply packages/dbml-parse/src/lib/analyzer/validator/elementValidators/table.ts:137]: yeah, `nameFragment` is mutated and `maybeNameFragments` is reused below so I cloned it.

**Review by NQPhuc — APPROVED** [review]
LGTM!

**Pull request merged**